gemrb/GUIScripts/Spellbook.py

```diff
@@ -97,6 +97,17 @@ def GetKnownSpells(actor, BookType):
 
 	return knownSpells
 
+def GetKnownSpellsDescription(actor, BookType):
+        """ Gets text to display in the chargen spell listing. """
+	info = ""
+        Spells = GetKnownSpells(actor, BookType)
+        # reverse spells order grouped by SpellLevel
+        Spells.sort(lambda fst, snd: -1 if fst['SpellLevel'] == snd['SpellLevel'] else 0)
+        for Spell in Spells:
+            SpellName = Spell['SpellName']
+            info += GemRB.GetString (SpellName) + "\n"
+	return info
+
```

Comment:
this bit is all sorts of misindented.

Reply:
Okay, I configured my editor to not expand tabs…

Reply:
```suggestion
def GetKnownSpellsDescription(actor, BookType):
	""" Prepares known and memorized spell listing for chargen. """

	info = ""
	Spells = GetKnownSpells(actor, BookType)
	# reverse spells order grouped by SpellLevel
	Spells.sort(lambda fst, snd: -1 if fst['SpellLevel'] == snd['SpellLevel'] else 0)
	for Spell in Spells:
		info += GemRB.GetString (Spell['SpellName']) + "\n"
	return info
```
Trying to fix it it for you via web, so I can merge sooner.

Reply:
you undid my fix, so you have to do it again.

Reply:
Oh, sorry. Forgot to pull your changes. Converted these again to tabs.